### ISSUE TITLE
Empty tag bug

### ISSUE THREAD

**Issue by cavivie** [maxmindlin/telegraf-rust#4]
Considering below code:
```rust
use telegraf::*;

#[derive(Metric)]
struct MyMetric {
    field: i32,
}

fn main() {
    let mut client = Client::new("udp://127.0.0.1:8080").unwrap();
    let point = MyMetric { field: 12345 };
    client.write(&point);
}
```

Run the code, and the telegraf agent process will get error: 
`Unable to parse incoming packet: metric parse error: expected tag at 1:13: "MyMetric, field=12345i"`

If I change the packet `"MyMetric, field=12345i"` to `"MyMetric field=12345i"`, it will work well, should the reason be extra comma?

Please check the telegraf documents: [line protocol](https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_tutorial/)

**Comment by maxmindlin** [comment]
Thanks for the issue! Interesting how I never caught this. It is correct that a LP with no tag set should not have the comma. When I get a moment I can look into this or will accept a PR

**Comment by maxmindlin** [comment]
This should be fixed in v0.4.4. Please reach out if this is not the case.

Thanks!

**Comment by cavivie** [comment]
Thanks very much
